bb8/src/internals.rs

```diff
@@ -272,6 +274,14 @@ impl AtomicStatistics {
                 .fetch_add(wait_time.as_micros() as u64, Ordering::SeqCst);
         }
     }
+
+    pub(crate) fn record(&self, kind: StatsKind) {
+        let stats = match kind {
+            StatsKind::ClosedBroken => &self.connections_closed_broken,
+            StatsKind::ClosedInvalid => &self.connections_closed_invalid,
+        };
+        stats.fetch_add(1, Ordering::SeqCst);
```

Comment:
Sorry, I meant you can chain the `fetch_add()` directly onto the `match { .. }` without binding it.